CONTRIBUTING.md

```diff
@@ -21,6 +21,7 @@ Known bugs and feature requests are all in the [issue tracker](https://github.co
 * [Fork](https://github.com/ggez/ggez/fork) and clone the repository
 * Create a new branch: git checkout -b my-branch-name
 * Make your changes
+> It's adviced to run [rustfmt](https://github.com/rust-lang-nursery/rustfmt) and [clippy](https://github.com/rust-lang-nursery/rust-clippy) before submitting a pull request
```

Comment:
Adding clippy is a bad idea because there are changes in existing lints and new lints every now and then, also sometimes it makes mistakes and/or complaints about absolutely sane things. Will add rustfmt soon.